ctapipe/reco/ImPACT.py

```diff
@@ -479,7 +480,7 @@ def get_likelihood(self, source_x, source_y, core_x, core_y,
                                                self.spe,
                                                self.ped[tel_count])
             if np.any(prediction == np.inf):
-                print("inf found at ", self.type[tel_count], zenith,
+                print("inf found at ", self.tel_type[tel_count], zenith,
```

Comment:
can you change this to use the logger, like `self.log.debug("info found at type=%s zenith=%.2f azimuth=%.2f energy=%.2f impact=%.2f x_max=%.2d", self.tel_type[tel_count], zenith, azimuth, energy, impact, x_max_bin)`